Sources/Transifex/Cache.swift

```diff
@@ -402,11 +412,11 @@ public final class TXStandardCache: TXDecoratorCache {
     ///
     /// - Parameters:
     ///   - overridePolicy: The specific override policy to be used when updating the internal
-    ///   memory cache with the stored contents from disk. Defaults to .overrideAll.
+    ///   memory cache with the stored contents from disk. Defaults to .overrideUsingTranslatedOnly.
     ///   - groupIdentifier: The group identifier of the app, if the app makes use of the app groups
     /// entitlement. Defaults to nil.
     @objc
-    public init(overridePolicy: TXCacheOverridePolicy = .overrideAll,
+    public init(overridePolicy: TXCacheOverridePolicy = .overrideUsingTranslatedOnly,
```

Comment:
As discussed offline, let's use `.overrideAll` as the default policy.

Reply:
Switched back to `.overrideAll` policy here.